packages/synapse-react-client/src/components/GenericCard/SynapseCardLabel.tsx

```diff
@@ -99,6 +100,12 @@ export const SynapseCardLabel: React.FC<SynapseCardLabelProps> = props => {
     return <Linkify text={str} className={newClassName} />
   }
 
+  if ('resolveEntityName' in labelLink && labelLink.resolveEntityName && str) {
+    const { baseURL, URLColumnName } = labelLink
+    const href = `/${baseURL}?${URLColumnName}=${str}`
+    return <EntityLink entity={str} link={href} />
```

Comment:
Let's hide the icon by default since we aren't linking to Synapse

```suggestion
    return <EntityLink entity={str} link={href} showIcon={false} />
```